synapse/cortex.py

```diff
@@ -5874,38 +5874,45 @@ async def _getMirrorProxy(self, opts):
         if self.stormpool is None:  # pragma: no cover
             return None
 
-        if self.stormpool.size() == 0:
+        size = self.stormpool.size()
+        if size == 0:
             logger.warning('Storm query mirror pool is empty, running query locally.')
             return None
 
-        proxy = None
+        for i in range(size):
```

Comment:
```suggestion
        for _ in range(size):
```

nit. take it or leave it. i can't find `i` being used.